Sources/NIO/Interfaces.swift

```diff
@@ -94,6 +97,12 @@ public final class NIONetworkInterface {
             self.pointToPointDestinationAddress = nil
         }
 
+        if (caddr.ifa_flags & UInt32(IFF_MULTICAST)) != 0 {
+            multicastSupported = true
+        } else {
+            multicastSupported = false
```

Comment:
Nit: can you use the explicit `self` keyword here? That's the preferred NIO style.